### DISCUSSION TITLE
Login: This user is blocked

### DISCUSSION THREAD

**Discussion by zeus-bloom** [vercel/community#5590] [Account Issue] [answered]
### Question

I am unable to login post signup. The moment i signed up - it said - This user is blocked. I am using github authentication.
Please help.

**Comment by zeus-bloom** [comment]
The problem up top is I can't get to support without signing in.

**Comment by amyegan** [comment, marked as answer]
Hey @zeus-bloom. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps.

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

To open a ticket please use the appropriate direct channel found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Comment by acarmagnani** [comment]
I have this same issue and can't find suppor either.
